src/CoreBundle/Resources/config/services.xml

```diff
@@ -11,6 +11,7 @@
         <parameter key="chameleon_system_core.database.estimation_lower_limit">0</parameter>
         <parameter key="chameleon_system_core.controller_invoke_listener.class">ChameleonSystem\CoreBundle\Controller\ChameleonControllerInvokeListener</parameter>
         <parameter key="chameleon_system_core.tpl_page_controller.class">TPkgCmsRouteControllerCmsTplPage</parameter>
+        <parameter key="chameleon_system_media_manager.render_media_properties_field">true</parameter>
```

Comment:
This is a parameter of chameleon_system_core and only modified in the media manager. Could you set the prefix to chameleon_system_core?

Reply:
You are right. Renamed the parameter.